recipes/perl-encode/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# If it has Build.PL use that, otherwise use Makefile.PL
+if [ -f Build.PL ]; then
+    perl Build.PL
+    perl ./Build
+    perl ./Build test
+    # Make sure this goes in site
+    perl ./Build install --installdirs site
+elif [ -f Makefile.PL ]; then
+    # Make sure this goes in site
+    perl Makefile.PL INSTALLDIRS=site
```

Comment:
I'm unsure about this. The proposed conda-forge Perl documentation at https://github.com/conda-forge/conda-forge.github.io/pull/1790 suggests `INSTALLDIRS=vendor`, and that's what I've been using for all recipes I've migrated. What do you think?

Reply:
To be honest, I don't know Conda well enough to judge which paths are most appropriate. If you consider `vendor` to be the better option, I will use that. I took these files unchanged from the `conda skeleton cpan` output.